src/auditor/dynamic_linkage.jl

```diff
@@ -184,18 +184,34 @@ function should_ignore_lib(lib, ::ELFHandle)
         "libthr.so.3",
         "libpthread.so.0",
     ]
+    if Sys.isfreebsd(platform)
+        push!(ignore_libs,
+              # From FreeBSD SDK
+              "libdevstat.sos.7",
+              "libexecinfo.so.1",
+              "libkvm.so.7",
+              "libz.so.6",
```

Comment:
I'm not 100% sure this is a good idea, as we may want to ensure that `libz.so.6` comes from our Zlib_jll` sometimes?